### ISSUE TITLE
请问你用的cuda版本是多少啊？

### ISSUE THREAD

**Issue by zyx1213271098** [hanxuhfut/CVPR2019_Underexposed_Photo_Enhancement_Using_Deep_Illumination_Estimation#1]

**Comment by hanxuhfut** [comment]
Tested on Python 2.7, Ubuntu 14.0, gcc-4.8，cuda 8.0，cudnn 5.1

**Comment by hanxuhfut** [comment]
> _No description provided._

Tested on Python 2.7, Ubuntu 14.0, gcc-4.8，cuda 8.0，cudnn 5.1

**Comment by hanxuhfut** [comment]
@zyx1213271098 
Tested on Python 2.7, Ubuntu 14.0, gcc-4.8，cuda 8.0，cudnn 5.1

**Comment by zyx1213271098** [comment]
Yes, after change to Python 2.7, Ubuntu 14.0, gcc-5.3，cuda 8.0，cudnn 5.1，I make success. Thanks
